docs/scripts.rst

```diff
@@ -0,0 +1,57 @@
+*******
+Scripts
+*******
+
+Overview
+========
+
+This document describes scripts that come with the Repo.
+
+frb_summary
+===========
+
+This script prints a simple summary of a given FRB and its
+host galaxy (when that exists) to the screen.
+
+Here is the usage::
+
+    usage: frb_summary [-h] [--verbose] frb_name
+
+    Script to print a summary of an FRB to the screen [v1.0]
+
+    positional arguments:
```

Comment:
Make apparent that "FRB" is case sensitive here in case you don't want to change the script as suggested in the comments on `frb_summary.py`.

Reply:
have made it so that it isn't.